dask_geomodeling/geometry/field_operations.py

```diff
@@ -571,8 +571,9 @@ class Where(BaseSingleSeries):
 
     Args:
       source (SeriesBlock): Source SeriesBlock that is going to be updated
-      cond (SeriesBlock): Conditional (boolean) SeriesBlock that determines
-        whether features in the source SeriesBlock will be updated.
+      cond (SeriesBlock): Conditional SeriesBlock that determines
+        whether features in the source SeriesBlock will be updated. Non-
+        boolean data will be converted using data -> True, nodata -> False.
```

Comment:
Will the audience understand that nodata / data refers to "being a 'missing value' (NaN, NaT or None)? We are nor not using nodata / data in a context of seriesblocks elsewhere.